test/support/factories/gtfs_factory.ex

```diff
@@ -48,7 +48,9 @@ defmodule Skate.GtfsFactory do
           "vehicle" => %{
             "id" => "G-10098",
             "label" => "3823-3605",
-            "license_plate" => nil
+            "license_plate" => nil,
+            "state_of_charge_percentage" => 85_800,
```

Comment:
Related to the other comment: can we change these millipercent values to a rounded integer percent?

```suggestion
            "state_of_charge_percentage" => 86
```

There are a few more to change in this PR, like `80_000`, and `value: 75000`